src/data/javascript-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const javascriptQuiz = [
+  {
+    Question:
+      "In Javascript, combination of function and its lexical environment is called?",
+    Answer: "Closure",
+    Distractor1: "Debouncing",
+    Distractor2: "Throttling",
+    Distractor3: "Scope Chaining",
+    Explanation: "A closure is the combination of a function bundled together (enclosed) with references to its surrounding state (the lexical environment). In other words, a closure gives you access to an outer function's scope from an inner function.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures"
```

Comment:
MDN is a good resource. 
But when in doubt always go for a freeCodeCamp resource and if one doesn't exist on the topic, then reach for MDN. 
The goal is to promote more of freeCodeCamp's articles and youtube videos first. 👍 

```suggestion
    Link: "https://www.freecodecamp.org/news/a-basic-guide-to-closures-in-javascript-9fc8b7e3463e/"
```

Reply:
agreed, thanks for the inputs.